src/jdk.accessibility/windows/native/toolscommon/AccessInfo.cpp

```diff
@@ -97,8 +97,7 @@ void logString(FILE *logfile, const char *msg, ...) {
     va_start(argprt, msg);
     vsnprintf(tmpbuf, sizeof(tmpbuf), msg, argprt);
 
-    fprintf(logfile, tmpbuf);
-    fprintf(logfile, "\n");
+    fprintf(logfile, "%s\n", tmpbuf);
```

Comment:
Why can't we use [`fputs`](https://en.cppreference.com/w/c/io/fputs.html) and avoid all the format string logic and ambiguity altogether?

Alternatively, *use `vfprintf` directly* instead of first formatting the arguments into a temporary buffer with `vsnprintf` and then sending the result to the output.

Reply:
After reading about fputs, I think this is best. Like you mentioned, no ambiguity since it does not use specifiers. Updated.

Reply:
Since Damon already made the change and its just one line I'm fine to keep it but not go looking for more.